sql/privilege.go

```diff
@@ -206,8 +240,8 @@ func (p *PrivilegeDescriptor) CheckPrivilege(user string, priv privilege.Kind) b
 		return false
 	}
 	// ALL is always good.
-	if userPriv.Privileges&(1<<privilege.ALL) != 0 {
+	if userPriv.Privileges&privilege.ALL.Mask() != 0 {
```

Comment:
The pattern `privilegeBits&privilegeKind.Mask() != 0` is super common. Perhaps make `privilegeBits` an explicit type (i.e. `type Bits uint32`) and give it a `Has()` method.